pymc3/__init__.py

```diff
@@ -34,3 +34,21 @@
     _log.setLevel(logging.INFO)
     handler = logging.StreamHandler()
     _log.addHandler(handler)
+
+def tqdm(*args, **kwargs):
+    try:
+        from tqdm import tqdm_notebook
+        return tqdm_notebook(*args, **kwargs)
+    except:
```

Comment:
I guess what we really want is https://github.com/tqdm/tqdm/issues/234. The particular error message just means that there is no kernel running. In the console backend we would still have a kernel, so I guess the error would be different.